obspy/core/tests/test_inventory.py

```diff
@@ -574,6 +574,19 @@ def test_inventory_can_be_initialized_with_no_arguments(self):
 
         self.assertEqual(inv, inv2)
 
+    def test_copy(self):
+        """
+        Test for copying inventory.
+        """
+        inv = read_inventory()
+        inv2 = inv.copy()
+        self.assertIsNot(inv, inv2)
+        self.assertEqual(inv, inv2)
+        # make sure changing inv2 doesnt affect inv
+        original_latitude = inv2[0][0][0].latitude
+        inv2[0][0][0].latitudue = original_latitude + 1
```

Comment:
latitudue  - latitude

Reply:
not sure how to understand this test...

Reply:
It just checks whether the memory copy works by modifying the copied reference and checking it doesn't touch the original reference.

Reply:
um.. but the latitude is never actually changed anywhere.. so i agree with @ThomasLecocq

Reply:
I mean the typo has to be fixed.. but the point is to prove that the two objects don't reference the same point in memory

Reply:
Fixed the typo already and also amended the test